src/server/util/auth/passport.init.js

```diff
@@ -0,0 +1,12 @@
+const passport = require('passport');
+const GithubStrategy = require('passport-github').Strategy;
+
+const { GITHUB_CONFIG } = require('../../config');
```

Comment:
Require vs imports again. Is there a way to use absolute paths instead of relative ones?

Reply:
Potentially, I'm making an issue for it and we can look into it in the future. I don't this to be a blocking problem